packages/base/src/formbuilder/formselectors.ts

```diff
@@ -1,14 +1,14 @@
 import { LayerType, SourceType } from '@jupytergis/schema';
-import { GeoJSONSourcePropertiesForm } from './objectform/geojsonsource';
-import { GeoTiffSourcePropertiesForm } from './objectform/geotiffsource';
-import { HeatmapLayerPropertiesForm } from './objectform/heatmapLayerForm';
-import { HillshadeLayerPropertiesForm } from './objectform/hillshadeLayerForm';
-import { LayerPropertiesForm } from './objectform/layerform';
-import { PathBasedSourcePropertiesForm } from './objectform/pathbasedsource';
-import { TileSourcePropertiesForm } from './objectform/tilesourceform';
-import { VectorLayerPropertiesForm } from './objectform/vectorlayerform';
-import { WebGlLayerPropertiesForm } from './objectform/webGlLayerForm';
-import { SourcePropertiesForm } from './objectform/sourceform';
+import { GeoJSONSourcePropertiesForm } from './objectform/sourceform/geojsonsource';
```

Comment:
There should be an `index.ts` in those sub-directories that export everything, simplifying the imports.

```typscript
import { GeoJSONSourcePropertiesForm, GeoTiffSourcePropertiesForm } from './forms/sourceforms';
```